notebooks/work-in-progress/explore_fuel_allocation.ipynb

```diff
@@ -2,8 +2,8 @@
  "cells": [
```

Comment:
<p>Our abbreviation for percentage has been "pct," but this isn't a percentage, it's a proportion / fraction.</p><p> </p><p>I think you can do this same check (that all three of these columns are close to each other) just using <code>np.isclose()</code> directly, and setting your own <code>rtol</code> (relative tolerance) and <code>atol</code> (absolute tolerance) values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>